ext/node/ops/fs.rs

```diff
@@ -78,3 +79,156 @@ where
   fs.cp_async(path, new_path).await?;
   Ok(())
 }
+
+#[derive(Debug, Serialize)]
+/// The fields are stored as strings because they could be bigint / u64, and
+/// precision may be lost if they are parsed to numbers by serde_v8.
+pub struct StatFs {
+  #[serde(rename = "type")]
+  pub typ: String,
+  pub bsize: String,
+  pub blocks: String,
+  pub bfree: String,
+  pub bavail: String,
+  pub files: String,
+  pub ffree: String,
```

Comment:
Use `u64`. serde_v8 will serialize it has BigInt if it doesn't fit. Source: https://docs.rs/serde_v8/latest/src/serde_v8/ser.rs.html#440-449